src/check.ts

```diff
@@ -20,6 +20,7 @@ export const check = async (input: lib.Input) => {
   core.setOutput("result", s);
   core.info(`result: ${s}`);
   const alreadyLocked = metadata?.state === "lock";
+  core.setOutput("locked", alreadyLocked);
   core.setOutput("already_locked", alreadyLocked);
```

Comment:
Honestly, I want to remove `already_locked` if `mode` is `check`, but it's a breaking change.